src/lib.rs

```diff
@@ -8,7 +8,6 @@
 // latest bitflags triggers a rustc bug with cross-crate macro expansions causing dead_code
 // warnings even though the macro expands into something with allow(dead_code)
 #![allow(dead_code)]
-#![deny(warnings)]
```

Comment:
Probably, because it wouldn't compile otherwise on whatever was the current nightly then. I'll recheck it, when I look at it again.